bin/conjur-cli/commands/server.rb

```diff
@@ -85,6 +88,16 @@ def load_bootstrap_policy
       ) || exit(($CHILD_STATUS.exitstatus))
     end
 
+    def cleanup_pidfile
```

Comment:
I would suggest adding a larger comment above the method describing why this is needed (the error observed), and why this solution fixes it.